pkg/vsphere/vm/vm.go

```diff
@@ -121,6 +121,22 @@ func (vm *VirtualMachine) FolderName(ctx context.Context) (string, error) {
 	return path.Base(u.Path), nil
 }
 
+// Folder returns a reference to the parent folder that owns the vch
```

Comment:
This isn't only for the VCH, so reword that it's the parent folder for the vm

Reply:
yeah good point. Fixing this